testdata/cheats/Fork2.t.sol

```diff
@@ -130,8 +113,7 @@ contract ForkTest is DSTest {
 
         cheats.selectFork(optimismFork);
         // this doesn't work yet because `dummy` is only available on `mainnetFork`
-        cheats.expectRevert();
-        dummy.hello();
+        // dummy.hello();
```

Comment:
should this be commented out?

Reply:
deleted since no longer used in this test, reverts with a meaningful message are now tested separately